addons/doc/static/src/components/doc_modal_search.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+<t t-name="web.DocSearchModal">
+    <div class="modal-bg flex justify-content-center align-items-center">
```

Comment:
beware that if the screen has a reduced height (by spawning the debug tools below) the modal's top may get out of the screen

Reply:
In general, the interface doesn't fit well in responsive mode